app/Kernel.php

```diff
@@ -153,6 +154,10 @@ protected function buildContainer()
             }
         }
 
+        $backendExtension = new BackendExtension();
+        $container->registerExtension($backendExtension);
+        $extensions[] = $backendExtension->getAlias();
```

Comment:
you could still change

```
$backendExtension = new BackendExtension();
$container->registerExtension($backendExtension);
```

to `$container->registerExtension(new BackendExtension());`